src/views/Profile/AdminTable/AdminDatePicker.tsx

```diff
@@ -65,7 +65,7 @@ export const AdminDatePicker: React.FC<IMaterialsDate> = ({
               onChange={(value) =>
                 setChanges({
                   option: 'start',
-                  date: value && value.toISOString(),
+                  date: value && value.toISOString().substring(0, 10),
```

Comment:
I propose to introduce helper in /src/utilities/formatDate.ts for date formating

1. shorter code
2. can cover with tests(also for edge cases)
3. consistent formatting across different occurences

do you agree?

Reply:
Create function helper for this date request